### PR TITLE
Update ArgoCD Integration.md

### PR DESCRIPTION
**Pull request by vainkop**
1. Update the doc since it is possible to use helm secrets in Argo CD for external Charts.
2. `sops` backend was tested & is working.

**What this PR does / why we need it**:
Update the doc since it is possible to use helm secrets in Argo CD for external Charts & the solution is in the same ArgoCD Integration.md [here](https://github.com/jkroepke/helm-secrets/blob/c6888bec3e0b96933b3ced1a7d8e6267ca4c5976/docs/ArgoCD%20Integration.md?plain=1#L114)

### CODE REVIEW

**Comment by jkroepke** [comment]
Good to hear this.

I guess, if it works, we can also remove the BETA from the headline? What did you mean?

**Comment by vainkop** [comment]
> Good to hear this.
> 
> I guess, if it works, we can also remove the BETA from the headline? What did you mean?

It works with sops 3.8.1 + ArgoCD 3.11.3 + AWS KMS but I haven't tested it against other providers & methods.

File: docs/ArgoCD Integration.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ because it is [registered](https://github.com/jkroepke/helm-secrets/blob/4e61c55
 
 ## Multi-Source Application Support [BETA]
```
Comment (jkroepke) [inline docs/ArgoCD Integration.md:72]:
```suggestion
## Multi-Source Application Support
```

Lets removed that BETA thing

**Review by jkroepke — APPROVED** [review]
Thanks!

**Pull request merged**